packages/x-components/tests/e2e/cucumber/common-steps.spec.ts

```diff
@@ -214,7 +221,6 @@ Given('a results API with a known response', () => {
           ...createFilter('Muñecas', false, createFilter => [
             ...createFilter('Peluches', false),
             ...createFilter('Ropa y accesorios', false),
-            ...createFilter('Muñecas', false),
```

Comment:
Why did you remove this filter?

Reply:
Because it was a duplicated filter which was causing an infinite loop issue. Filters are indexed by its id, and this filter has the same id  than its parent. You can imagine the rest of the story.